PedelecApp/src/de/tum/in/www1/PedelecApp.java

```diff
@@ -47,15 +47,109 @@ public void start(Stage primaryStage) {
         primaryStage.show();
 		
 		reserveButton.setOnAction(new EventHandler<ActionEvent>() {
-		    @Override public void handle(ActionEvent e) {
-		        System.out.println("Reserve Button clicked");
-		        
-		        Reservation reservation = new Reservation();
-		        reservation.setBike(pedelecNametext.getText());
-		        reservation.setStartDate(datePicker.getValue());
-		        reservation.setStartTime(timeTextField.getText());
-		        reservation.save();
-		    }
+		    @Override
+public
+void
+ handle(ActionEvent 
+e
+) {
+System.
+out
+.println(
+"Reserve Button clicked"
+);
+LocalDate 
+XX1
+ = 
+datePicker
+.getValue();
+String 
+x_1
+ = 
+timeTextField
+.getText(), 
+x_2
+ = 
+x_1
+, 
+x_3
+;
+x_2
+ = 
+pedelecNametext
+.getText();
+            String 
+x_4
+ = 
+x_2
+;
+x_3
+ = 
+"Please confirm your reservation of "
+ + 
+x_2
+ + 
+" at "
+ + 
+XX1
+.format(DateTimeFormatter.
+ISO_LOCAL_DATE
+) + 
+" "
+ + 
+timeTextField
+.getText();
+  ButtonType 
+x__4
+ = 
+new
+ Alert(AlertType.
+CONFIRMATION
+, 
+""
+ + 
+x_3
+).showAndWait().get();
+if
+ (
+x__4
+ == ButtonType.
+OK
+) {
+Reservation 
+r33
+ = 
+new
+ Reservation(), 
+r34
+;
+r33
+.setBike(
+x_4
+);
+r33
+.setStartDate(
```

Comment:
It is not possible to understand the code at all.. better use an IDE instead of the text editor.